roles/splunk/tasks/configure_firewall.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Ensure {{ firewall_service }} package is installed
+  ansible.builtin.package:
+    name: "{{ firewall_service }}"
+    state: present
+  become: true
+
+- name: Configure firewalld for Splunk
+  block:
+    - name: Ensure firewalld is Started and Enabled
```

Comment:
I changed this to say `firewalld` instead of `firewall_service` because it was a bit confusing in the output when it ran on Debian. It showed `skipped` and name said `ufw`.